hyundai_kia_connect_api/Vehicle.py

```diff
@@ -284,6 +292,26 @@ def ev_estimated_station_charge_duration(self, value):
         self._ev_estimated_station_charge_duration_unit = value[1]
         self._ev_estimated_station_charge_duration = value[0]
 
+    @property
+    def ev_target_range_charge_AC(self):
+        return self._ev_target_range_charge_AC
```

Comment:
The setter sets 3 properties: _ev_target_range_charge_AC_value, _ev_target_range_charge_AC_unit and _ev_target_range_charge_AC

This getter only returns one of these properties. Is it intentional or are the 2 other getters missing?

---

Overall, I'm not a fan of this getter/setter use, I find it a little too hard to read/use.

My recommendation would be to just have 2 public attributes and do away with the getters/setters:
- ev_target_range_charge_AC_value
- ev_target_range_charge_AC_unit

Reply:
I am thinking that may be the way to go.    That being said I am going to merge this as is since all range items follow this.   A refactor should clean up the whole thing but I want to keep consistency for now.